program-test/src/lib.rs

```diff
@@ -805,7 +808,7 @@ impl ProgramTest {
         debug!("Payer address: {}", mint_keypair.pubkey());
         debug!("Genesis config: {}", genesis_config);
 
-        let mut bank = Bank::new_with_runtime_config_for_tests(
```

Comment:
Should we also delete `Bank::new_with_runtime_config_for_tests()`, since this was the only call site? It's funny that I was looking at this function just yesterday to see if it should be DCOU.